.travis.yml

```
@@ -171,16 +158,16 @@ jobs:
  
  # Clippy linting.
   - name: clippy
-    rust: nightly-2018-10-23
+    rust: nightly-2018-11-27
     install:
-    - cargo clippy --version | rustup component add clippy-preview --toolchain=nightly-2018-10-23
+    - cargo clippy --version | rustup component add clippy-preview --toolchain=2018-11-27
```

Is _pipe_ (`|`) here intentional? Shouldn't it be

```suggestion
    - cargo clippy --version || rustup component add clippy-preview --toolchain=2018-11-27
```

which installs clippy if it is not already installed?